### ISSUE TITLE
NO_AUTHORS evidence is non-string type (used by OpenNeuro)

### ISSUE THREAD

**Issue by nellh** [bids-standard/legacy-validator#1414]
This field throws an error when the result is passed to OpenNeuro and results in validation hanging.

<details>
<summary>Example --json output</summary>

```json
{
  "issues": {
    "errors": [],
    "warnings": [
      {
        "key": "NO_AUTHORS",
        "severity": "warning",
        "reason": "The Authors field of dataset_description.json should contain an array of fields - with one author per field. This was triggered because there are no authors, which will make DOI registration from dataset metadata impossible.",
        "files": [
          {
            "key": "NO_AUTHORS",
            "code": 113,
            "file": null,
            "evidence": [],
            "line": null,
            "character": null,
            "severity": "warning",
            "reason": "The Authors field of dataset_description.json should contain an array of fields - with one author per field. This was triggered because there are no authors, which will make DOI registration from dataset metadata impossible.",
            "helpUrl": "https://neurostars.org/search?q=NO_AUTHORS"
          }
        ],
        "additionalFileCount": 0,
        "helpUrl": "https://neurostars.org/search?q=NO_AUTHORS",
        "code": 113
      },
      {
        "key": "README_FILE_SMALL",
        "severity": "warning",
        "reason": "The recommended file /README is very small. Please consider expanding it with additional information about the dataset.",
        "files": [
          {
            "key": "README_FILE_SMALL",
            "code": 213,
            "file": null,
            "evidence": null,
            "line": null,
            "character": null,
            "severity": "warning",
            "reason": "The recommended file /README is very small. Please consider expanding it with additional information about the dataset.",
            "helpUrl": "https://neurostars.org/search?q=README_FILE_SMALL"
          }
        ],
        "additionalFileCount": 0,
        "helpUrl": "https://neurostars.org/search?q=README_FILE_SMALL",
        "code": 213
      }
    ],
    "ignored": []
  },
  "summary": {
    "sessions": [],
    "subjects": [
      "SENSETestNov20"
    ],
    "subjectMetadata": [
      {
        "participantId": "SENSETestNov20",
        "age": 44,
        "sex": "M"
      }
    ],
    "tasks": [],
    "modalities": [
      "MRI"
    ],
    "secondaryModalities": [
      "MRI_Structural"
    ],
    "totalFiles": 6,
    "size": 10260169,
    "dataProcessed": false,
    "pet": null,
    "dataTypes": [
      "T1w"
    ]
  }
}```

</details>

**Comment by nellh** [comment]
This was fixed in #1417
